src/ucs/config/ucm_opts.c

```diff
@@ -25,9 +32,12 @@ static ucs_config_field_t ucm_global_config_table[] = {
    "Enable memory events",
    ucs_offsetof(ucm_global_config_t, enable_events), UCS_CONFIG_TYPE_BOOL},
 
-  {"MMAP_RELOC", "yes",
-   "Enable installing mmap symbols in the relocation table",
-   ucs_offsetof(ucm_global_config_t, enable_mmap_reloc), UCS_CONFIG_TYPE_BOOL},
+  {"MMAP_HOOK_MODE", "bistro",
+   "MMAP hook mode\n"
+   " none   - don't set mmap hooks.\n"
+   " reloc  - use ELF relocation table to set hooks.\n"
+   " bistro - use binary instrumentation to set hooks.\n",
```

Comment:
We have to document this somewhere, like wiki. It would be useful to provide usecases.